source/puppet/6.4/configuration.md

```diff
@@ -799,7 +799,7 @@ directories readable by Puppet Server when necessary.
 
 The hiera configuration file. Puppet only reads this file on startup, so you must restart the puppet master every time you edit it.
 
-- *Default*: $confdir/hiera.yaml. However, if a file exists at $codedir/hiera.yaml, Puppet uses that instead.
+- *Default*: $codedir/hiera.yaml. For backwards compatibility $confdir/hiera.yaml will be used if $codedir/hiera.yaml is missing.
```

Comment:
There's no change in 6.4 regarding codedir, so this should still be the same as the previous files:

*Default*: $confdir/hiera.yaml. However for backwards compatibility, if a file exists at $codedir/hiera.yaml, Puppet uses that instead.

Reply:
you are right. I don't know exactly how I messed that up. How ever, it's good you noticed it. :+1:

Reply:
how exactly do I accept your change?

Reply:
I'm not used to github and PR. Did I accept your change by clicking resolve conversation?

Reply:
now I think it's fixed. At least it looks like fixed for me in the "Files changed" tab.